### ISSUE TITLE
Authorization HTTP header is inserted as parameter even though there is securityScheme defined

### ISSUE THREAD

**Issue by ipepe** [exoego/rspec-openapi#160]
I have defined securitySchemes as follows:
```yaml
components:
  securitySchemes:
    ApiKeyAuth:
      type: apiKey
      in: header
      name: AUTHORIZATION
```

But when I'm generating a schema file, seems like `AUTHORIZATION` header is being constantly suggested as parameter:

```yaml
      parameters:
      - name: AUTHORIZATION
        in: header
        required: true
        schema:
          type: string
        example: AuthorizationExampleSecretKey
```

From my understanding this isn't necessary or even contradicts to OpenAPI specification. Can we somehow parse `securitySchemes` for headers that should not be added as parameters?

**Comment by exoego** [comment]
According to https://swagger.io/docs/specification/authentication/, we can refer to security schemes in `security` property.
And there is no need to include such security parameters in the `parameters` property.

```yaml
components:
  securitySchemes:
    ApiKeyAuth:
      type: apiKey
      in: header
      name: X-API-Key

paths:
  /billing_info:
    get:
      summary: Gets the account billing info
      security:
        - ApiKeyAuth
      # parameters: [] 
      # ^.. no need!! 
      responses:
        '200':
          description: OK
```

Such handling is not implemented.
PR welcome.

**Comment by exoego** [comment]
Implemented in #166
